src/entities/user.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'users' })
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({
+    nullable: false,
+  })
+  email: string;
+
+  @Column({
+    nullable: false,
+  })
+  hashed_password: string;
```

Comment:
(suggestion; non-blocking): We should consider using consistent casing. In this case, converting these snack case property names to camel case.